src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -160,6 +200,7 @@ public void writeThrough(User user) {
     userEntity.setProperty("password", user.getPassword());
     userEntity.setProperty("about", user.getAbout());
     userEntity.setProperty("creation_time", user.getCreationTime().toString());
+    userEntity.setProperty("isAdmin", user.getIsAdmin());
```

Comment:
If we're using snake case for properties above (i.e. creation_time), isAdmin better be is_admin for consistency